src/main/java/org/zalando/nakadi/domain/Version.java

```diff
@@ -0,0 +1,51 @@
+package org.zalando.nakadi.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public class Version {
+    private final int major;
```

Comment:
I agree that it is more clear to have versions as integers but from other side you never use them in the code and all these housekeeping work requires resources + we claim that we use `semantic versioning` which support string at patch level. We can just use String inside that object and extend it if necessary, wdyt?